pr-data.csv

```diff
@@ -6379,7 +6377,7 @@ https://github.com/soot-oss/soot,23758df0ce8327bb414d982aeb325a4f5e83ccf8,.,soot
 https://github.com/soot-oss/soot,23758df0ce8327bb414d982aeb325a4f5e83ccf8,.,soot.toolkits.graph.TestDominance.TestSimpleDiamond,ID,Accepted,https://github.com/soot-oss/soot/pull/1802,
 https://github.com/spdx/tools,cf965bc2fa99e1c5ca23f992f18cef48c08dec9f,.,org.spdx.compare.TestPackageSheet.tesConcludedLicensetCol,ID,Accepted,https://github.com/spdx/tools/pull/312,
 https://github.com/spdx/tools,cf965bc2fa99e1c5ca23f992f18cef48c08dec9f,.,org.spdx.compare.TestPackageSheet.testDeclaredLicenseCol,ID,Accepted,https://github.com/spdx/tools/pull/312,
-https://github.com/spdx/tools,cf965bc2fa99e1c5ca23f992f18cef48c08dec9f,.,org.spdx.rdfparser.license.TestLicenseInfoFactory.testParseSPDXLicenseStringMixedCase,ID,,,
+https://github.com/spdx/tools,cf965bc2fa99e1c5ca23f992f18cef48c08dec9f,.,org.spdx.rdfparser.license.TestLicenseInfoFactory.testParseSPDXLicenseStringMixedCase,ID,Unmaintained,,latest commit 2025/10/07
```

Comment:
2025/10/17 is just a few days ago -- if you want to claim that a project is `Unmaintained`, the latest commit should be at least two years ago.